domain/user/bootstrap/bootstrap.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2023 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package bootstrap
+
+import (
+	"context"
+
+	"github.com/canonical/sqlair"
+	"github.com/juju/errors"
+
+	"github.com/juju/juju/core/database"
+	"github.com/juju/juju/domain/user/state"
+)
+
+// InsertAdminUser inserts the admin user into database.
```

Comment:
The Admin user is a core bootstrap concept and business logic. I feel like we should explain a little more of what its purpose is for and why we create it. That is to say can we explain the why and what of the admin user so that future Juju changes understand it's importance.

Reply:
I've extended the comment message.